app/src/main/res/layout/topic_lessons_fragment.xml

```diff
@@ -12,7 +12,7 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/oppia_background">
+    android:background="@color/component_color_topic_revision_fragment_background_color">
```

Comment:
@iampradeep-hr Why is this change in this PR? This PR is focused on revision card only, right? If this change is needed then do add screenshot of topic-lessons too.

Reply:
It was not having dark mode, seems someone is already working on it. so i will drop it and revert back.